### PR TITLE
[Fix](SparkLoad): fix the timeout aborted loadtasks are not cleaned up.

### LINKED ISSUE #15446: [Bug] sparkload push tasks are not cleaned up when txn timeout aborted
### Search before asking

- [X] I had searched in the [issues](https://github.com/apache/doris/issues?q=is%3Aissue) and found no similar issues.

### Version

From 0.13 to the newest

### What's Wrong?

When the transaction of spark load is timeout and aborted, the push tasks are not cleaned up and alway kept  in AgentTaskQueue.

This will cause useless push tasks are always scheduled in BE.

### What You Expected?

Clear spark load job, when spark load's transaction is timeout and aborted.

### How to Reproduce?

1. set `spark_load_default_timeout_second` to 60s
2. start a heavy spark load job, which exceeds 60s
3. wait for transaction timeout and abort
4. find out whether the push tasks of the aborted job are still kept in BE

### Anything Else?

_No response_

### Are you willing to submit PR?

- [X] Yes I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by spaces-X**
# Proposed changes
close #15446 

## Problem summary

Describe your changes.

## Checklist(Required)

1. Does it affect the original behavior: 
    - [ ] Yes
    - [x] No
    - [ ] I don't know
2. Has unit tests been added:
    - [ ] Yes
    - [ ] No
    - [x] No Need
3. Has document been added or modified:
    - [ ] Yes
    - [ ] No
    - [x] No Need
4. Does it need to update dependencies:
    - [ ] Yes
    - [x] No
5. Are there any changes that cannot be rolled back:
    - [ ] Yes (If Yes, please explain WHY)
    - [ ] No

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java [hunk 1]
```diff
@@ -695,6 +695,8 @@ private void clearJob() {
             // clear job infos that not persist
             sparkLoadAppHandle = null;
             resourceDesc = null;
+            etlOutputPath = "";
+            appId = "";
```
Comment (wangbo) [inline fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java:699]: Why set empty string here?

Comment (spaces-X) [reply fe/fe-core/src/main/java/org/apache/doris/load/loadv2/SparkLoadJob.java:699]: If the job is cancelled by user, the `clearJob()` will be called twice.
Set thess shtring empty,to ensure the idempotency of the `clearJob` function.

**Comment by hello-stephen** [comment]
TeamCity pipeline, clickbench performance test result:
 the sum of best hot time: 35.7 seconds
 load time: 652 seconds
 storage size: 17123647190 Bytes
 https://doris-community-test-1308700295.cos.ap-hongkong.myqcloud.com/tmp/20221229113123_clickbench_pr_70925.html

**Review by wangbo — DISMISSED** [review]
LGTM

**Comment by spaces-X** [comment]
rebase the newest master branch for pipeline failures.

**Review by wangbo — APPROVED** [review]
LGTM

**Pull request merged**